src/resources/MachineLearning/SmartSnippetsConfiguration/SmartSnippetsConfigurationInterfaces.ts

```diff
@@ -59,28 +59,42 @@ export interface SmartSnippetsDocumentGroupPreviewParams {
 export interface SmartSnippetsDocumentGroupPreview {
     /**
      * The query that was used to fetch document information.
+     *
+     * @Example @source==("Salesforce Notifier")
      */
     query: string;
     /**
-     * Status indicating whether there are enough candidates for learning.
+     * The total number of documents in the selected sources.
      */
-    documentRequirementStatus: SmartSnippetsDocumentRequirementStatus;
+    numberOfDocuments: string;
```

Comment:
This used to be a `number` here. It also looks like to be a number according to the Swagger Docs. Anything I'm missing?

Reply:
Its fixed. Thanks!